project_liberation/settings/base.py

```diff
@@ -181,3 +182,6 @@
 }
 
 STATICFILES_STORAGE = 'pipeline.storage.PipelineStorage'
+
+# A key for Google API, necessary for accessing the map on the mainpage.
+GOOGLE_API_KEY = ''
```

Comment:
@maciejSamerdak one thing if you add such setting like this please keep it commented. When it will be commented it will force developer to add such a setting in `local_settings` instead of thinking why he has in settings google api key with wrong length (which information is now given by `manage.py check` when it is not commented out from `base.py`). I will fix it